src/android/OpenTokAndroidPlugin.java

```diff
@@ -383,6 +383,13 @@ public void onVideoEnabled(SubscriberKit arg0, String arg1) {
             // TODO Auto-generated method stub
 
         }
+
+        public void unsubscribe(JSONArray args) {
+            mSession.unsubscribe(mSubscriber);
+            ViewGroup frame = (ViewGroup) cordova.getActivity().findViewById(android.R.id.content);
+            frame.removeView( this.mView );
```

Comment:
This may also cause an issue because you'll be removing the subscriber view from the JS thread and not the thread that the view was created on